src/components/page-header/page-header.component.ts

```diff
@@ -37,7 +39,25 @@ export class PageHeaderComponent {
     @Input() backVisible: boolean = true;
 
     /** The alignment of the secondary navigation tabs. */
-    @Input() secondaryNavigationAlignment: 'left' | 'right' | 'center' = 'center';
+    _secondaryNavigationAlignment: NavigationAlignment;
+
+    @Input()
+    set secondaryNavigationAlignment(value: NavigationAlignment) {
+        this._secondaryNavigationAlignment = value;
+    }
+
+    get secondaryNavigationAlignment(): NavigationAlignment {
+        let alignment = this._secondaryNavigationAlignment;
+
+        if (
+            (alignment === 'left' && this.secondaryNavigationLeadingContentTemplate) ||
+            (alignment === 'right' && this.secondaryNavigationTrailingContentTemplate)
+         ) {
+            alignment = 'center';
```

Comment:
I don't think we decided for force center alignment if templates are provided. Check out how I did it with the primary navigation in combination with the subtitle - this is doable with CSS only. (The ticket mentions this.)